torchrl/modules/planners/common.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import abc
+from typing import Optional
+
+import torch
+
+from torchrl.data.tensordict.tensordict import TensorDictBase
+from torchrl.envs import EnvBase
+from torchrl.modules import TensorDictModule
+
+__all__ = ["MPCPlannerBase"]
+
+
+class MPCPlannerBase(TensorDictModule, metaclass=abc.ABCMeta):
+    """
+    MPCPlannerBase Module. This is an abstract class and must be implemented by the user.
+
+    This class inherits from TensorDictModule. Provided a TensorDict, this module will perform a Model Predictive Control (MPC) planning step.
+    At the end of the planning step, the MPCPlanner will return the action that should be taken.
+
+    Args:
+        env (Environment): The environment to perform the planning step on (Can be ModelBasedEnv or EnvBase).
```

Comment:
What's an Environment here?
Don't forget the "`" when you're pointing to existing classes